sale_invoice_group_method/README.rst

```diff
@@ -0,0 +1,102 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License LGPL-3
+
+=========================
+Sale Invoice Group Method
+=========================
+
+This module allows you to combine several Sales Orders into a single invoice,
+if they meet the split criteria defined by the 'Invoice Group Method'.
+
+The split criteria is defined in the 'Invoice Group Method' by a combination
+of fields of the Sales Order. 'Invoice Address', 'Currency' and 'Payment Term'
+are always included.
+
+You can assign a default 'Invoice Group Method' to customers, so that it will
+be proposed on their orders.
+
+When no Invoice Group Method is defined in a Sales Order, the standard
+approach will be used, which groups by 'Invoice Address' and 'Currency'.
+
+Note: Existing draft invoices are not considered in the process of grouping.
+However, you can find the feature implemented in ``sale_merge_draft_invoice``
```

Comment:
Hmmm, recommending a module that isn't available for the version? I get it, just saying, is meant to be user documentation afterall.

Reply:
It is available here: https://github.com/OCA/sale-workflow/pull/569. It's just that it's waiting for more reviews in order to be merged :)

Reply:
Other reviewers, please ignore above, it is absolutely non blocking.